eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,49 +1,50 @@
 param (
```

Comment:
Given we are changing the parameters this will break the c++ usages so we will need to figure out how to stage that correctly.

Reply:
This is how it was broken last time: https://github.com/Azure/azure-sdk-for-cpp/issues/3061
That will be the scenario to validate isn't broken when we make these changes.

Reply:
Removed the get-codeowner script and release the tool change first to make sure we did not break c++ use case